### PR TITLE
release build_web_compilers v0.4.2

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: build_web_compilers/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## 0.4.2
 
-- Add magic comment marker for build_runner to know where to inject 
+- Add magic comment marker for build_runner to know where to inject
```
Comment (kevmoo) [inline build_web_compilers/CHANGELOG.md]: nit: could we be a bit more clear here? Where is it added? In which contexts?

Comment (jakemac53) [reply build_web_compilers/CHANGELOG.md]: I don't know that it is really relevant for users of the package (beyond the fact that its now supported)?

It is added in the main *.dart.js script though.

Comment (jakemac53) [reply build_web_compilers/CHANGELOG.md]: ill add a note that this is only supported with the `dartdevc` compiler

**Comment by jakemac53** [comment]
Looks like a new analyzer breakage with the latest dev release - submitting on red for now.

**Pull request merged**